src/tools/scripts/incoherenceStructure/structure_avec_coselec.js

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env node
+'use strict';
+
+const { execute } = require('../../utils');
+const path = require('path');
+const fs = require('fs');
+
+const getStructure = async db => await db.collection('structures').find({
+  'statut': 'VALIDATION_COSELEC'
+}).toArray();
+
+const countMisesEnRelation = db => async idStructure => await db.collection('misesEnRelation').countDocuments({
+  'structure.$id': idStructure,
+  'statut': { '$in': ['finalisee', 'recrutee', 'renouvellement_initiee'] }
+});
+
+execute(__filename, async ({ logger, db, exit }) => {
+  logger.info('Début de la recherche d\'incohérence sur le nombre de conseillerslorsqu\'une structure a un statut VALIDATION_COSELEC');
+  const promises = [];
+  const writeLine = [];
+
+  const structures = await getStructure(db);
+  structures.forEach(structure => {
+    promises.push(new Promise(async resolve => {
+      await countMisesEnRelation(db)(structure._id).then(countNbConseiller => {
+        if (countNbConseiller !== structure.coselec[structure.coselec.length - 1].nombreConseillersCoselec) {
```

Comment:
il peut y a voir une diff si on met `'renouvellement_initiee'` car il peut y avoir pour un finalisee & un autre statut `'renouvellement_initiee'` ce qui veut dire ça compte 2 aux lieux de 1 
A voir si on fait juste ça => `'statut': { '$in': ['finalisee', 'recrutee'] }`